project-koku/koku
# [COST-3009] Get OCP on Cloud Unallocated Cost

## Jira Ticket

[COST-3009](https://issues.redhat.com/browse/COST-3009)

## Description

This change will add in the necessary capacity fields for OCP on Cloud cost calculations on the Unallocated projects. 

## Testing

1. Checkout Branch
2. make create-test-customer
3. make load-test-customer-data
4. WAIT
5. Hit http://127.0.0.1:8000/api/cost-management/v1/reports/openshift/infrastructures/gcp/costs/?filter[time_scope_value]=-1
6. Hit http://127.0.0.1:8000/api/cost-management/v1/reports/openshift/infrastructures/gcp/costs/?group_by[project]=*filter[time_scope_value]=-1
7. See the same total cost in the meta for each endpoint (group by project used to be less)
8. Repeat those API calls subbing gcp for aws/azure
9. 🎊 

## Notes

...
